logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -53,6 +72,10 @@ def close
   end
 end
 
+class DummyOutputMore < DummyOutput
```

Comment:
Can you explain to me probably the need of this dummy/mock plugin definition? ```DummyOutputMore``` ? not much clear to me the purpose as we have to mocks defined for this.